### PR TITLE
Update SetupGuide_Python.md

### PR DESCRIPTION
**Pull request by lucyzhang929**

### CODE REVIEW

File: docs/SetupGuide_Python.md [hunk 1]
```diff
@@ -115,23 +113,27 @@ Note: This tutorial requires that a SQL database is setup as shown in [Create a
...
-_TODO_
+In this case, the parameter value of the `@Name` parameter is an empty string.
+
+https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/GetProductsNameEmpty
 
 #### Null Parameter Value
 
+If the `{name}` specified in the `getproducts-namenull/{name}` URL is "null", the query returns all rows for which the Name column is `NULL`. Otherwise, it returns all rows for which the value of the Name column matches the string passed in `{name}`
+
 _TODO_
```
Comment (lucyzhang929) [inline docs/SetupGuide_Python.md:130]: Will be adding this sample in a follow up PR.

File: docs/SetupGuide_Python.md [hunk 2]
```diff
@@ -115,23 +113,27 @@ Note: This tutorial requires that a SQL database is setup as shown in [Create a
 
 #### Query String
 
-_TODO_
+The input binding executes the `SELECT * FROM Products WHERE Cost = @Cost` query. The Parameters argument passes the `{cost}` specified in the URL that triggers the function, `getproducts/{cost}`, as the value of the `@Cost` parameter in the query. CommandType is set to `Text`, since the constructor argument of the binding is a raw query.
```
Comment (Charles-Gagnon) [inline docs/SetupGuide_Python.md:116]: As I mentioned to Maddy in the other one - it'd be better I think to put these descriptions into the functions themselves and then just have this be the link to the function folder. That way people looking at the samples directly can benefit from the descriptions too.

File: docs/SetupGuide_Python.md [hunk 3]
```diff
@@ -115,23 +113,27 @@ Note: This tutorial requires that a SQL database is setup as shown in [Create a
...
 
 #### Stored Procedure
 
-_TODO_
+`SelectProductsCost` is the name of a procedure stored in the user's database. In this case, CommandType is `StoredProcedure`. The parameter value of the `@Cost` parameter in the procedure is once again the `{cost}` specified in the `getproducts-storedprocedure/{cost}` URL.
 
-#### IAsyncEnumerable
-
-_TODO_
+https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/GetProductsStoredProcedure
```
Comment (Charles-Gagnon) [inline docs/SetupGuide_Python.md:136]: nit: One thing I just learned about is that for "proper" markdown raw links like this should be put in <>
```suggestion
<https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/GetProductsStoredProcedure>
```

File: docs/SetupGuide_Python.md [hunk 4]
```diff
@@ -198,18 +200,14 @@ Note: This tutorial requires that a SQL database is setup as shown in [Create a
...
 ### Samples for Output Bindings
 
-#### ICollector&lt;T&gt;/IAsyncCollector&lt;T&gt;
-
-_TODO_
-
 #### Array
 
-_TODO_
+https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/AddProductsArray
```
Comment (Charles-Gagnon) [inline docs/SetupGuide_Python.md:205]:
```suggestion
<https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/AddProductsArray>
```

File: docs/SetupGuide_Python.md [hunk 5]
```diff
@@ -198,18 +200,14 @@ Note: This tutorial requires that a SQL database is setup as shown in [Create a
...
 
 #### Single Row
 
-_TODO_
+https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/AddProduct
```
Comment (Charles-Gagnon) [inline docs/SetupGuide_Python.md:209]:
```suggestion
<https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/AddProduct>
```

File: docs/SetupGuide_Python.md [hunk 6]
```diff
@@ -115,23 +113,27 @@ Note: This tutorial requires that a SQL database is setup as shown in [Create a
...
+The input binding executes the `SELECT * FROM Products WHERE Cost = @Cost` query. The Parameters argument passes the `{cost}` specified in the URL that triggers the function, `getproducts/{cost}`, as the value of the `@Cost` parameter in the query. CommandType is set to `Text`, since the constructor argument of the binding is a raw query.
+
+https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/GetProducts
 
 #### Empty Parameter Value
 
-_TODO_
+In this case, the parameter value of the `@Name` parameter is an empty string.
+
+https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/GetProductsNameEmpty
```
Comment (Charles-Gagnon) [inline docs/SetupGuide_Python.md:124]:
```suggestion
<https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/GetProductsNameEmpty>
```

File: docs/SetupGuide_Python.md [hunk 7]
```diff
@@ -115,23 +113,27 @@ Note: This tutorial requires that a SQL database is setup as shown in [Create a
 
 #### Query String
 
-_TODO_
+The input binding executes the `SELECT * FROM Products WHERE Cost = @Cost` query. The Parameters argument passes the `{cost}` specified in the URL that triggers the function, `getproducts/{cost}`, as the value of the `@Cost` parameter in the query. CommandType is set to `Text`, since the constructor argument of the binding is a raw query.
+
+https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/GetProducts
```
Comment (Charles-Gagnon) [inline docs/SetupGuide_Python.md:118]: nit: One thing I just learned about is that for "proper" markdown raw links like this should be put in <>

https://www.markdownguide.org/basic-syntax#urls-and-email-addresses

(let's try out this fancy batched suggestions too...)

```suggestion
<https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/GetProducts>
```

Comment (VasuBhog) [reply docs/SetupGuide_Python.md:118]: I would include something prior to the link however such as "Sample can be found [here](link)." (especially if we plan to just include links by themselves. 

That would just be clear and concise. Also would not have to include the long links potentially.

Comment (Charles-Gagnon) [reply docs/SetupGuide_Python.md:118]: Yeah, I like that. I'd suggest naming the link though - e.g. 

`See the [GetProducts](...) sample`

Although that sounds a bit incomplete to me...but I can't think of anything better myself so if you come up with anything feel free to throw it out as a suggestion and we can see which one we like the best.

**Comment by lucyzhang929** [comment]
Opened a [new PR](https://github.com/Azure/azure-functions-sql-extension/pull/677) to address comments - closing this one.

**Pull request closed without merging**